server/belga/macros/change_headline_and_urgency.py

```diff
@@ -0,0 +1,18 @@
+"""This macro adds a prefix string 'TIP' to headline and sets urgency to 5 of the item"""
+
+
+def change_headline_and_urgency(item, **kwargs):
+    if 'headline' in item:
+        item['headline'] = 'TIP ' + item.get('headline', '')
+
+    if 'urgency' in item:
```

Comment:
why this condition?

Reply:
Actually, I wasn't sure if we should add the urgency even if the item doesn't have it already.